substrate/frame/nomination-pools/src/tests.rs

```diff
@@ -3264,12 +3264,47 @@ mod pool_withdraw_unbonded {
 			CurrentEra::set(StakingMock::current_era() + StakingMock::bonding_duration() + 1);
 			assert_ok!(Pools::pool_withdraw_unbonded(RuntimeOrigin::signed(10), 1, 0));
 
-			// Then there unbonding balance is no longer locked
+			// Then their unbonding balance is no longer locked
 			assert_eq!(StakingMock::active_stake(&default_bonded_account()), Ok(15));
 			assert_eq!(StakingMock::total_stake(&default_bonded_account()), Ok(15));
 			assert_eq!(Balances::free_balance(&default_bonded_account()), 20);
 		});
 	}
+	#[test]
+	fn pool_withdraw_unbonded_creates_tvl_diff() {
+		ExtBuilder::default().add_members(vec![(20, 10)]).build_and_execute(|| {
+			// Given 10 unbond'ed directly against the pool account
+			assert_ok!(Pools::unbond(RuntimeOrigin::signed(20), 20, 5));
+
+			assert_eq!(StakingMock::active_stake(&default_bonded_account()), Ok(15));
+			assert_eq!(StakingMock::total_stake(&default_bonded_account()), Ok(20));
+			assert_eq!(Balances::free_balance(&default_bonded_account()), 20);
+			assert_eq!(TotalValueLocked::<T>::get(), 20);
+
+			// When
+			CurrentEra::set(StakingMock::current_era() + StakingMock::bonding_duration() + 1);
+			assert_ok!(Pools::pool_withdraw_unbonded(RuntimeOrigin::signed(10), 1, 0));
+			assert_eq!(TotalValueLocked::<T>::get(), 15);
+
+			let member_balance = PoolMembers::<T>::iter()
+				.map(|(_, member)| member.total_balance())
+				.reduce(|acc, total_balance| acc + total_balance)
+				.unwrap_or_default();
+
+			// Then their unbonding balance is no longer locked
+			assert_eq!(StakingMock::active_stake(&default_bonded_account()), Ok(15));
+			assert_eq!(StakingMock::total_stake(&default_bonded_account()), Ok(15));
+			assert_eq!(Balances::free_balance(&default_bonded_account()), 20);
+
+			// The difference between TVL and member_balance is exactly the difference between
+			// `total_stake` and the `free_balance`.
+			// This relation is not guaranteed in the wild as arbitrary transfers towards
```

Comment:
Any transfer that increases the free balance of the pool's bonded account that doesn't originate from the nom-pool pallet is not accounted for in the sum of `total_balance` of all pool members, nor in the TVL. 

If such a transfer would happen then free balance would be increased whereas sum of total_balance and TVL would remain the same. If we were now to show the relation as its done in this test it wouldn't work out because the difference between `total_stake` and `free_balance` is larger than the difference between `TVL` and sum of `total_balance` for all pool members. Hence the disclaimer that this test assumes things that may not hold up in the wild.